sdk/core/Azure.Core/api/Azure.Core.net6.0.cs

```diff
@@ -398,6 +400,18 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class Delay
```

Comment:
This is temporarily renamed to avoid collisions with the internal DelayStrategy which cannot be removed until after this PR is merged, and an Autorest update is made.

Reply:
Will it go back to being named `DelayStrategy` at that time?  Wondering if we considered `RetryDelay` as a type name, or if the delay is intended to be used more widely than retries?

Reply:
Yeah the final name is probably still TBD. It would be used in LROs as well so my thinking is that it is better to avoid using `Retry` in the name.